daprdocs/content/en/operations/monitoring/jaeger.md

```diff
@@ -0,0 +1,173 @@
+---
+type: docs
+title: "How-To: Set up Jaeger for distributed tracing"
+linkTitle: "Jaeger"
+weight: 3000
+description: "Set up Jaeger for distributed tracing"
+type: docs
+---
+Dapr currently supports two kind of tracing protocol: OpenCensus and
+Zipkin. Since Jaeger is compatible with Zipkin, we can use the Zipkin
+protocol to talk to Jaeger.
+
+## Configure self hosted mode
+
+### Setup
+
+For self hosted mode, on running `dapr init` the following YAML files are created by default and they are referenced by default on `dapr run` calls unless otherwise overridden.
+
+1. The following file in `$HOME/.dapr/components/zipkin.yaml` or `%USERPROFILE%\dapr\components\zipkin.yaml`:
+
+* **zipkin.yaml**: Update the port 9411 to 9412 in this file, since
+often Zipkin is already installed and listen on 9411 in the self
+hosted mode. Recall that we are using the Zipkin protocol to talk to
+Dapr, so the exporter type in the file below is `exporters.zipkin`
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: zipkin
+  namespace: default
+spec:
+  type: exporters.zipkin
+  version: v1
+  metadata:
+  - name: enabled
+    value: "true"
+  - name: exporterAddress
+    value: "http://localhost:9412/api/v2/spans"
+```
+2. The following file in `$HOME/dapr/config.yaml` or `%USERPROFILE%\dapr\config.yaml`:
```

Comment:
What is the action the reader needs to do at this stage? is it to create this file and copy the contents below? please clarify
Also, it would better to clarify that the "or" refers to the OS you are running. Maybe something like this: 
`$HOME/dapr/config.yaml` (on Linux/MacOS) or `%USERPROFILE%\dapr\config.yaml` (on Windows)

Reply:
I copied from zipkin.md, which may have more explanations to tell the users where the component files are.

Updated now to just create a new component file and use `--components-path` instead.